Makefile

```diff
@@ -181,6 +184,12 @@ chplcheck: frontend-shared FORCE
 	@# writing this target is always FORCEd (so we'd end up building it twice).
 	cd tools/chplcheck && $(MAKE) all install
 
+lint-standard-modules: chplcheck FORCE
+	chplcheck --skip-unstable \
```

Comment:
Probably not an issue with this PR (and you could consider it future work), but we probably should have a flag for `--skip-deprecated` too.